ts_src/psetv2/pset.ts

```diff
@@ -65,26 +63,44 @@ export class Pset {
     return pset;
   }
 
-  static ECCKeysGenerator(ec: Secp256k1Interface): KeysGenerator {
+  static ECCKeysGenerator(ecc: {
+    pointFromScalar(
+      privateKey: Uint8Array,
+      compressed?: boolean,
+    ): Uint8Array | null;
```

Comment:
we should give this interface a name so it can be reused in tests, and get rid of importing TinySecp256Interface/Ecc interface there.

Reply:
renamed `KeysGeneratorSecp256k1Interface` (generate also the private key and returns a `KeysGenerator`)